_includes/menu.html

```diff
@@ -2,7 +2,7 @@
 <header class="navbar navbar-default navbar-demo navbar-fixed-top eslint-nav" id="top" role="banner">
   <div class="container">
 
-    <a href="/" class="navbar-brand"><img alt="ESLint" src="/img/logo.svg" itemprop="image">ESLint</a>
+    <a href="{{ site.url }}/" class="navbar-brand"><img alt="ESLint" src="{{ site.url }}/img/logo.svg" itemprop="image">ESLint</a>
```

Comment:
But all of those path are starting with `/` so if you forked the whole repo, you probably have all of the assets in that repo as well, right? I guess all of us always just used local install of Jekyll to work on the site, and never tried it in a fork.